product_docs/docs/pgd/5/reference/functions.mdx

```diff
@@ -232,7 +232,9 @@ Alias for `bdr.get_consensus_status`.
 #### Synopsis
 
 ```sql
-bdr.raft_leadership_transfer(IN node_name text, IN wait_for_completion boolean)
+bdr.raft_leadership_transfer(node_name text, 
+                             wait_for_completion boolean, 
+                             node_group text DEFAULT NULL)
```

Comment:
I believe the 3rd option is actually `node_group_name`.  I believe this because that is the function that actually existed in PGD5.3 in BA, running EDB Postgres Extended.  
![image](https://github.com/EnterpriseDB/docs/assets/83652362/3f8bea51-c143-4ab1-886c-595602ec74af)